tests/rptest/scale_tests/high_throughput_test.py

```diff
@@ -26,6 +26,7 @@
                                       MetricsEndpoint, SISettings)
 from rptest.tests.prealloc_nodes import PreallocNodesTest
 from ducktape.mark import ok_to_fail
+import re
```

Comment:
nit: would prefer this import line be further up in the file along the other import lines placed in alphabetical order